packages/react/src/components/Tabs/next/Tabs.js

```diff
@@ -194,26 +314,124 @@ function TabList({
     }
   });
 
+  useLayoutEffect(() => {
```

Comment:
I think you'll need to use our `useIsomorphicEffect` utility for this to be safe for SSR